### ISSUE TITLE
Vite 5 support?

### ISSUE THREAD

**Issue by amr3k** [bluwy/vite-plugin-iso-import#8]
I see this warning:

```js
 WARN  Issues with peer dependencies found
.
└─┬ vite-plugin-iso-import 1.0.0
  └── ✕ unmet peer vite@"^2.0.0 || ^3.0.0 || ^4.0.0": found 5.0.10
```

Does this plugin support vite 5?

**Comment by bluwy** [comment]
Yeah it should still work in Vite 5. I can bump the peer dep range.

**Comment by bluwy** [comment]
Released v1.1.0 👍

**Comment by amr3k** [comment]
Wow , that was quick.
Appreciate your efforts, thanks.
